### PR TITLE
Added realtime + roi ideas for sunpy

### PR DESCRIPTION
**Pull request by dpshelio**

### CODE REVIEW

File: _projects/2017/sunpy/realtimesun.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+name: RealTime solar web.
```
Comment (Cadair) [inline _projects/2017/sunpy/realtimesun.md]: Needs a more catch title ;)

File: _projects/2017/sunpy/realtimesun.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+desc: Back & Front end for a Real Time solar viewer
```
Comment (Cadair) [inline _projects/2017/sunpy/realtimesun.md]: needs a better description (this is the line or two that sells the project).

File: _projects/2017/sunpy/realtimesun.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# add a description of your project
+requirements:
+ - Familiarity with Python
+ - Understanding of databases
+ - Basic web development
```
Comment (Cadair) [inline _projects/2017/sunpy/realtimesun.md]: some familiarity with Flask?

File: _projects/2017/sunpy/roi.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ - sunpy
+---
+
+#### Description
+
+This project aims to plan and implement a object that could be used to
+define a region of interest and interact with the rest of data types used
+in sunpy.
+
+For example, a region of interest could be a particular are in the sun
```
Comment (Cadair) [inline _projects/2017/sunpy/roi.md]: typo: "area" also Sun not sun

File: _projects/2017/sunpy/roi.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+This project aims to plan and implement a object that could be used to
+define a region of interest and interact with the rest of data types used
+in sunpy.
+
+For example, a region of interest could be a particular are in the sun 
+on a particular moment in time. Therefore, that object should contain the limits
+of such area, its contour and the time when it was observed.
+It also needs other information, such as the coordinate system that it
+uses (notice that chaincode needs of a image pixel size too) and the wavelength
```
Comment (Cadair) [inline _projects/2017/sunpy/roi.md]: the section in brackets is meaningless to someone who doesn't know what a chaincode is, I would drop it.

File: _projects/2017/sunpy/roi.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+tags:
+ - sunpy
+collaborating_projects:
+ - sunpy
+---
+
+#### Description
+
+This project aims to plan and implement a object that could be used to
+define a region of interest and interact with the rest of data types used
```
Comment (Cadair) [inline _projects/2017/sunpy/roi.md]: I would describe more what you mean my ROI, i.e.: a region of interest, in any physical dimension, such as spatial area, temporal or spectral window.

File: _projects/2017/sunpy/realtimesun.md (hunk 3 above)
Comment (dpshelio) [inline _projects/2017/sunpy/realtimesun.md]: Required in the following line

**Pull request merged**